R/quosure-errors.R

```diff
@@ -0,0 +1,2 @@
+# needed to handle try-e
+setOldClass("quosure.error")
```

Comment:
```suggestion
setOldClass("quosure.error")

```

(or move to a new file)